spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/BaseReader.java

```diff
@@ -252,7 +252,13 @@ protected class SparkDeleteFilter extends DeleteFilter<InternalRow> {
     private final InternalRowWrapper asStructLike;
 
     SparkDeleteFilter(String filePath, List<DeleteFile> deletes, DeleteCounter counter) {
-      super(filePath, deletes, table.schema(), expectedSchema, counter);
+      super(
+          filePath,
+          deletes,
+          table.schema(),
+          expectedSchema,
+          DeleteFilter.DEFAULT_STREAM_FILTER_THRESHOLD,
+          counter);
```

Comment:
This is the only change I make to spark/v3.2 here, which is needed because the superclass, `DeleteFilter`, now takes the additional parameter.
Once this PR is merged, I can port the changes to spark/v3.3 to v3.2.